### PR TITLE
Changelog: Noting 1.0-beta-7

### PR DESCRIPTION
**Pull request by fcojfernandez**
Just an incremental Beta release with recent changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+Bugfixes and improvements:
+
+- [[JENKINS-54352](https://issues.jenkins-ci.org/browse/JENKINS-54352)] Add functional tests
+- [[JENKINS-56079](https://issues.jenkins-ci.org/browse/JENKINS-56079)] Print the classpath in case of passing the wrong directories to Bootstrap
+
+Internal changes:
+
+* [[PR #26](https://github.com/jenkinsci/jenkinsfile-runner/pull/26)] Allow to pass worfklow job parameters
+* [[PR #64](https://github.com/jenkinsci/jenkinsfile-runner/pull/64)] Add README with instructions for running tests on macOS
+* [[PR #69](https://github.com/jenkinsci/jenkinsfile-runner/pull/6)] Check for null in workflow parameters
```
Comment (varyvol) [inline CHANGELOG.md]: I'm not sure I'd put this one, given the functionality is really the same than the first internal change.

**Review by varyvol — COMMENTED** [review]
Just a small comment.

**Review by varyvol — APPROVED** [review]

**Comment by fcojfernandez** [comment]
The CI is failing and the message we are receiving is:
```
Error response from daemon: conflict: unable to delete 48dd0ab99acc (cannot be forced) - image is being used by running container f9b448838c51
ASSERT:Unknown failure encountered running a test
```
This PR is nothing to do with this error, so I'll merge it and open a PR to fix it

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+## [1.0-beta-7] - 2019-02-20
+
+Bugfixes and improvements:
+
+- [[JENKINS-54352](https://issues.jenkins-ci.org/browse/JENKINS-54352)] Add functional tests
+- [[JENKINS-56079](https://issues.jenkins-ci.org/browse/JENKINS-56079)] Print the classpath in case the wrong directories are provided to Bootstrap
+
+Internal changes:
+
+* [[PR #26](https://github.com/jenkinsci/jenkinsfile-runner/pull/26)] Allow to pass worfklow job parameters
```
Comment (oleg-nenashev) [inline CHANGELOG.md:12]: It is not an internal change. It is actually a major feature. Also `--no-sandbox` could be mentioned

**Pull request merged**